alert-database/src/main/java/com/synopsys/integration/alert/database/notification/NotificationContentConverter.java

```diff
@@ -33,11 +33,11 @@
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Component;
 
-import com.synopsys.integration.alert.TypeConverter;
 import com.synopsys.integration.alert.common.ContentConverter;
 import com.synopsys.integration.alert.common.data.model.Config;
+import com.synopsys.integration.alert.common.data.model.NotificationConfig;
 import com.synopsys.integration.alert.database.DatabaseEntity;
-import com.synopsys.integration.alert.database.notification.NotificationContent;
+import com.synopsys.integration.alert.database.TypeConverter;
 
 @Component
 public class NotificationContentConverter extends TypeConverter {
```

Comment:
Added a jira issue to remove the parent TypeConverter class in the future as this is the only class that uses TypeConverter now